.github/workflows/spelling.yml

```diff
@@ -1,154 +1,52 @@
+    env:
+      UPLOAD_SARIF_LIMITED: ''
     steps:
-    - name: check-spelling
-      id: spelling
-      uses: check-spelling/check-spelling@2c9e4a85102fa9b6df3cb8bb5a8dc8bdc2fb2fea # v0.0.23
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       with:
-        suppress_push_for_open_pull_request: 1
-        checkout: true
-        check_file_names: 1
-        spell_check_this: rancher-sandbox/rancher-desktop@main
-        post_comment: 0
-        use_magic_file: 1
-        experimental_apply_changes_via_bot: ${{ github.repository_owner != 'rancher-sandbox' && 1 }}
-        report-timing: 1
-        warnings: bad-regex,binary-file,deprecated-feature,large-file,limited-references,no-newline-at-eof,noisy-file,non-alpha-in-dictionary,token-is-substring,unexpected-line-ending,whitespace-in-dictionary,minified-file,unsupported-configuration,no-files-to-check
-        use_sarif: ${{ (!github.event.pull_request || (github.event.pull_request.head.repo.full_name == github.repository)) && 1 }}
-        extra_dictionary_limit: 20
-        extra_dictionaries:
-          cspell:software-terms/dict/softwareTerms.txt
-          cspell:k8s/dict/k8s.txt
-          cspell:node/dict/node.txt
-          cspell:aws/aws.txt
-          cspell:golang/dict/go.txt
-          cspell:php/dict/php.txt
-          cspell:python/src/python/python-lib.txt
-          cspell:typescript/dict/typescript.txt
-          cspell:npm/dict/npm.txt
-          cspell:shell/dict/shell-all-words.txt
-          cspell:html/dict/html.txt
-          cspell:filetypes/filetypes.txt
-          cspell:fullstack/dict/fullstack.txt
-          cspell:python/src/common/extra.txt
-          cspell:java/src/java.txt
-          cspell:dotnet/dict/dotnet.txt
-          cspell:css/dict/css.txt
-          cspell:django/dict/django.txt
-          cspell:docker/src/docker-words.txt
-          cspell:cpp/src/stdlib-cmath.txt
-          cspell:python/src/python/python.txt
-          cspell:powershell/dict/powershell.txt
+        persist-credentials: false
 
-  update:
-    name: Update PR
-    permissions:
-      contents: write
-      pull-requests: write
-      actions: read
-    runs-on: ubuntu-latest
-    if: ${{
-        github.repository_owner != 'rancher-sandbox' &&
-        github.event_name == 'issue_comment' &&
-        github.event.issue.pull_request &&
-        contains(github.event.comment.body, '@check-spelling-bot apply')
-      }}
-    concurrency:
-      group: spelling-update-${{ github.event.issue.number }}
-      cancel-in-progress: false
-    steps:
-    - name: apply spelling updates
-      uses: check-spelling/check-spelling@2c9e4a85102fa9b6df3cb8bb5a8dc8bdc2fb2fea # v0.0.23
+    - uses: ./.github/actions/yarn-install
+    - uses: ./.github/actions/setup-environment
+    - run: sudo apt-get install cpanminus
+
+    - name: Check Spelling
+      run: yarn lint:spelling
+      env:
+        GITHUB_TOKEN: ${{ github.token }} # for SARIF uploading
```

Comment:
Isn't the serif uploading being done in the next step?

Reply:
Right, I need to change the comment to say it's for SARIF _generation_.